### DISCUSSION TITLE
How can I get the "build number"

### DISCUSSION THREAD

**Discussion by TetsuOu** [community/community#26370] [Actions]
<p>I’m new to Github Actions.<br>
My question is:<br>
What the number ‘5’ below calls in Github Actions ?<br>
I know that in .travis.yml of Travis CI it calls <code>$TRAVIS_BUILD_NUMBER</code> .<br>
Thank you!:smile:<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/b/8/b80016d46ca15f9cdedb249929fcb15ad5534ec5.png" data-download-href="/uploads/short-url/qfK2PohS1KadFDqMRPvdfIQeFmZ.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/49858361/181097366-60d5caf2-f809-424e-b789-1d7b7cab7f83.png" alt="image" data-base62-sha1="qfK2PohS1KadFDqMRPvdfIQeFmZ" width="690" height="257" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/b/8/b80016d46ca15f9cdedb249929fcb15ad5534ec5_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">837×312 15.4 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by TetsuOu** [comment, marked as answer]
<p>I’m so sorry that I have not read the <a href="https://docs.github.com/en/free-pro-team@latest/actions/reference/context-and-expression-syntax-for-github-actions" rel="noopener nofollow ugc"> Context and expression syntax for GitHub Actions</a> of Github Docs first.<br>
The solution is <code>${{github.run_number}}</code>.</p>

**Reply by rob00builder** [reply]
Thanks, for the answer and the page to look at! Yay!
